app/src/main/java/com/scottyab/challenge/di/Modules.kt

```diff
@@ -45,144 +46,157 @@ import org.koin.dsl.module
 import retrofit2.Retrofit
 import retrofit2.converter.moshi.MoshiConverterFactory
 
-val appModule = module {
-    single<ImageLoader> { CoilImageLoader() }
-    single { AppCoroutineScope() }
-    single<AndroidResources> { AndroidResourcesProvider(androidContext()) }
-    factory { SimpleThrottler() }
-    single { androidContext().getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager }
-    factory<LocationCalculator> { AndroidLocationCalculator() }
-}
-
-val networkModule = module {
-    single {
-        Moshi.Builder()
-            .add(KotlinJsonAdapterFactory())
-            .build()
```

Comment:
There's a formatting change here, switching between two macs. Not had time to look into the issue. In a real project I'd track this down and resolve as it this type of formating change adds noise to the PR review.